packages/components/src/radio-control/README.md

```diff
@@ -104,7 +104,7 @@ A function that receives the value of the new option that is being selected as i
 
 An array of objects containing the value and label of the options.
 
--   `label`: `string` The label to be shown to the user.
+-   `label`: `string` | `ReactNode` The label to be shown to the user.
```

Comment:
Removing `string` as it's included in `ReactNode`. Also adding accessibility guidance.

```suggestion
-   `label`: `ReactNode` The label to be shown to the user. When the label is not a string, make sure that the element is accessibly labeled.
- ```